script.js

```diff
@@ -1,31 +1,26 @@
 // each charge of the form [x, y, charge]
+let charges = [];
+let sliders = [];
 
+// create sliders to control the charges
 function askForPoints() {
   let points = window.prompt("How many points do you want to graph?");
-  sliders = [];
   for (var i = 0; i < points; i++) {
-    sliders.push(document.createElement('div'));
-    sliders.push([]);
-    createSlider(10, window.innerWidth - 10, i);
-    createSlider(10, window.innerHeight - 10, i);
-    createSlider(-200, 200, i);
-    document.body.appendChild(sliders[2*i]);
+    slidersForCharge(i);
   }
-  charges = [];
   for (let i = 0; i < points; i++) {
     let currentSliders = sliders[2*i + 1];
     charges.push([JSON.parse(currentSliders[0].value), JSON.parse(currentSliders[1].value), JSON.parse(currentSliders[2].value)]);
   }
-  console.log(charges);
 }
 
-// slider.oninput = function() {
-//   output.innerHTML = this.value;
-//   drawFieldLines(charges);
-// }
-
-window.onResize = function () {
-  drawFieldLines();
+function slidersForCharge(i) {
+  sliders.push(document.createElement('div'));
+  sliders.push([]);
```

Comment:
This data structure is a bit odd. It seems that you're putting a `div` in each odd element of `sliders` and then a list into which you push all the slider elements into the even elements. Not entirely clear what that's about but I suspect there's a simpler way to handle it.